zmon_worker_monitor/builtins/plugins/http.py

```diff
@@ -297,6 +297,35 @@ def prometheus(self):
 
         return samples_by_name
 
+    def prometheus_flat(self, include_keys=[]):
```

Comment:
Was wondering what if we added a param to ``prometheus()`` method: ``prometheus(flat=True)``

Reply:
another remark: Using ``include_keys=None`` instead of ``include_keys=[]`` is preferred in args.